src/humanize/number.py

```diff
@@ -205,23 +205,44 @@ def intword(value: NumberOrString, format: str = "%.1f") -> str:
     except (TypeError, ValueError):
         return str(value)
 
+    is_negative = value < 0
+
+    if is_negative:
+        value *= -1
+
     if value < powers[0]:
-        return str(value)
+        return "-" + str(value) if is_negative else str(value)
```

Comment:
For example:

```suggestion
        return negative_prefix + str(value)
```

And similarly below.